README.md

```diff
@@ -1,39 +1,62 @@
 # JupiterOne Repository Relationship Generator
 
-A script that generates JupiterOne entity relationships between a repository and its dependencies.
-This script will look for dependencies within a repository's `package.json` file as well as the `dependencies.yaml`file if the repository has a `deploy` directory.
+A script that generates JupiterOne entity relationships between a repository and
+its dependencies. This script will look for dependencies within a repository's
+`package.json` file as well as the `dependencies.yaml`file if the repository has
+a `deploy` directory.
+
+This script currently works with node.js dependencies in package.json only. It
+does not scan for devDependencies.
```

Comment:
This phrasing is a little confusing... Maybe something like "This script only creates relationships for `dependencies` (not `devDependencies`)". NBD though if you don't want to make another commit.

I'd add this comment as a suggestion so you could just adopt it if you wanted, but GitHub doesn't support multiline suggestions. :(